manifests/params.pp

```diff
@@ -43,14 +43,15 @@
           }
         }
         $service_name        = pick($::mongodb::globals::service_name, 'mongod')
-        $mongos_service_name = pick($::mongodb::globals::mongos_service_name, 'mongos')
-        $config              = '/etc/mongod.conf'
+        $mongos_service_name = pick($::mongodb::globals::mongos_service_name, 'mongos') 
+        $config              = '/etc/mongodb.conf'
```

Comment:
Are you sure about this? What version of mongo from which package source and on what version of redhat?